thingif/src/main/java/com/kii/thingif/clause/trigger/AndClauseInTrigger.java

```diff
@@ -23,18 +25,29 @@ public AndClauseInTrigger(@NonNull TriggerClause ...clauses){
 
     @Override
     public List<TriggerClause> getClauses() {
-        return this.clauses;
+        return new ArrayList<>(this.clauses);
     }
 
     @Override
-    public void addClause(@NonNull TriggerClause clause) {
+    public AndClauseInTrigger addClause(@NonNull TriggerClause clause) {
         this.clauses.add(clause);
+        return this;
     }
 
     @Override
     public JSONObject toJSONObject() {
```

Comment:
@shinsuke-nara I actually don't know whether `GSON` can meet our requirements or not yet.  I think it is better to make a decision later.